javascripts/main.js

```diff
@@ -43,26 +44,45 @@ client.listen('disconnect', function() {
 
 // listen for and answer incoming calls
 client.listen('call', function(evt) {
-  call = evt.call;
   
-  if (call.initiator !== true) {
-    call.answer({
-      onConnect: onConnect,
-      onLocalMedia: onLocalVideo
-    });
+  if (evt.call.caller !== true) {
+      if (evt.call.incomingMedia.hasScreenShare()) {
+          call = evt.call;
+          call.answer({
+              onConnect: onConnect,
+              onHangup: function() {
+                  call = null;
+                  $('#callControls').hide();
+              }
+          });
+      } else {
+          audioCall = evt.call;
+          audioCall.answer({
+              constraints: {
+                  audio: true,
+                  /* There's some kind of bug preventing video from working for the non-screensharer. This
```

Comment:
have we got an issue in Jira to track this issue? Is there an issue raised with the Chrome team?